tools/tertiary-analysis/seurat/seurat_dim_plot.xml

```diff
@@ -59,14 +53,10 @@
         <param type="float" name="pt_size" value='1' label="Point size" help='Adjust point size for plotting' />
         <param type="float" name="label_size" value='4' label="Label size" help='Sets size of labels'/>
         <param type="boolean" name="do_label" checked="false" label='Label clusters' truevalue="TRUE" falsevalue="FALSE" />
+        <param type="boolean" name="repel" checked="false" label='Repel labels' truevalue="TRUE" falsevalue="FALSE" />
         <param type="text" name="group_by" value='ident' label='Group by' help='Group (color) cells in different ways' />
-        <param type="text" name="plot_title" format="text" label='Plot title' />
-        <param type="boolean" name="do_bare" checked="false" label='Do only minimal formatting' truevalue="TRUE" falsevalue="FALSE" />
         <param type="text" name="cols_use" format="text" label="Colors" help='Comma-separated list of colors, each color corresponds to an identity class. By default, ggplot assigns colors.' />
         <param type="text" name="pt_shape" format="text" label="Cell attribute for shape" help='If NULL, all points are circles (default). You can specify any cell attribute (that can be pulled with FetchData) allowing for both different colors and different shapes on cells.' />
-        <param type="boolean" name="coord_fixed" checked="false" label="Fixed-scaled coordinate system" help='Use a fixed scale coordinate system (for spatial coordinates). Default is FALSE.' truevalue="TRUE" falsevalue="FALSE" />
-        <param type="boolean" name="no_axes" checked="false" label="Remove Axes" help='Setting to TRUE will remove the axes.' truevalue="TRUE" falsevalue="FALSE" />
-        <param type="boolean" name="dark_theme" checked="false" label='Use a dark theme for the plot..' truevalue="TRUE" falsevalue="FALSE" />
```

Comment:
Why were these parameters (as well as a couple more above) deleted?  Have they been removed in the new Seurat version?

Reply:
Most likely, in fairness this is work done more than a year ago :-(.